routers/repo/pull.go

```diff
@@ -615,8 +615,10 @@ func ParseCompareInfo(ctx *context.Context) (*models.User, *models.Repository, *
 	if isSameRepo {
 		headRepo = ctx.Repo.Repository
 		headGitRepo = ctx.Repo.GitRepo
+		ctx.Data["BaseName"] = headRepo.Owner.Name
 	} else {
 		headGitRepo, err = git.OpenRepository(models.RepoPath(headUser.Name, headRepo.Name))
+		ctx.Data["BaseName"] = headRepo.BaseRepo.Owner.Name
```

Comment:
Use `baseRepo.OwnerName` instead since `headRepo.BaseRepo.Owner` might be nil.

Reply:
updated